tests/acceptance/test_acceptance/test_datafile.py

```diff
@@ -0,0 +1,168 @@
+from tests.acceptance.helpers import ENDPOINT_DATAFILE
+from tests.acceptance.helpers import create_and_validate_request_and_response
+
+
+expected_response = {"accountId": "10845721364", "anonymizeIP": True,
```

Comment:
if possible can you place in a separate file

Reply:
yeah, was thinking about that